doc/qbk/02_2_strings.qbk

```diff
@@ -12,22 +13,66 @@
 [section:strings Using Strings]
 
 Modifiable sequences of characters are represented using instances of type
-[link json.ref.boost__json__string `string`].
-These strings have an interface almost identical to `std::string`, and
-operate the same way, with these variations:
+__string__.
+These strings have a modified __string_view__ based version of the `std::string` interface, and
+operate the same way, except that:
 
-* __string__ is not a class template.
+* __string__ is not a class template,
+
+* __string__ only uses `char` as its character type,
+
+* redundant overloads for string operations have been replaced
+with a __string_view__ based interface,
+
+* access to characters in the range `[size(), capacity())` is permitted, and
 
 * __storage__ is used instead of __Allocator__.
 
-* New characters are 
+With augmented interface, operations requiring an input string
+are implemented as a single overload with a parameter of type
+__string_view__, and can accept most string-like objects. 
+Objects such as null terminated character pointers, `std::string`,
+`json::string`, subranges of strings, and objects
+convertible to __string_view__ can all be passed to these functions. 
 
-A string may be constructed initially empty without incurrening any
-memory allocations using the default storage, or a specified storage:
+[snippet_strings_4]
 
-[snippet_strings_1]
+More formally, `std::string` member function overloads that accept any of the
+following parameter combinations as an input string:
+
+* a `std::string` parameter, or
+
+* a `std::string` parameter and two `size_type` parameters that specify a substring, or
```

Comment:
> two `size_type` parameters that specify a substring

Two numbers can specify substring or range of string in at least two different ways.

Reply:
The docs are a work in progress, I imagine that we will continue to keep improving them over many pull requests. @sdkrystian

Reply:
@vinniefalco Understood. I just picked and followed the nitpicking approach.

Reply:
@mloskot  There's a careful balance between being pedantic and not boring the user. I could write these in standardese, but that would bore the user. Besides, this specification doesn't need to be specific as it outlines the overloads that are removed, so exact behavior is not needed.

Reply:
@sdkrystian Yes, the balance is important but AFAIU that part is "formal spec" where such balance does not quite apply (as it doesn't in API reference either). Anyway, I'm being anal and I expect a [function spec to be boringly precise](https://en.cppreference.com/w/cpp/string/basic_string/substr). Anyway, my question has been answered. Thx.

Reply:
The place for boring precision is in the javadoc. The exposition should be concise, elegant, and inspiring.